recipes/libstudxml/all/conanfile.py

```diff
@@ -0,0 +1,124 @@
+from conans import ConanFile, AutoToolsBuildEnvironment, MSBuild, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+required_conan_version = ">=1.33.0"
+
+class LibStudXmlConan(ConanFile):
+    name = "libstudxml"
+    description = "A streaming XML pull parser and streaming XML serializer implementation for modern, standard C++."
+    topics = ("xml", "xml-parser", "serialization")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://www.codesynthesis.com/projects/libstudxml/"
+    license = "MIT"
+    settings = "os", "compiler", "build_type", "arch"
+
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+    }
+
+    _autotools = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+
+    def requirements(self):
+        self.requires("expat/2.4.1")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version],
+                  destination=self._source_subfolder, strip_root=True)
+
+    def _configure_autotools(self):
+        if not self._autotools:
+            args = ["--with-extern-expat", "CXXFLAGS=\"-UNDEBUG\""]
```

Comment:
https://github.com/windoze/libstudxml/blob/e34acc39f93526e6ff9ade7539600539be6f5055/configure.ac#L35-L39
```suggestion
            args = ["--with-external-expat", "CXXFLAGS=\"-UNDEBUG\""]
```

Reply:
Huh... their INSTALL file is wrong apparently.
> One configure option worth mentioning is --with-extern-expat. It makes
libstudxml use an external Expat library rather than bulding-in the
internal version.